pages/profiles/tsimamada.md

```diff
@@ -0,0 +1,5 @@
+* #### My name is Rasolo niaina
+* I'm from **Madagascar**
+* _I work at_ [Minister of National Education][site men]
```

Comment:
Seems like this link would not work without adding `http://` to the url on line 5